ppr-ui/src/components/tables/mhr/SearchedResults.vue

```diff
@@ -341,6 +341,11 @@ export default defineComponent({
         return result.includeLienInfo !== true ? { ...result, includeLienInfo: false } : result
       })
       localState.totalResultsLength = resp.totalResultsSize
+      if (localState.searchType === UISearchTypes.MHR_NUMBER && localState.totalResultsLength === 1) {
+        const val = true
+        // Select search result if an MHR Number Search and search results equals 1.
+        localState.results = localState.results.map(result => ({ ...result, selected: val }))
```

Comment:
if `val` isn't use anywhere else in its scope, does it make sense to apply the value directly to the mapped property?
ie `selected: true`

Reply:
Good idea... I will make the change...